css/types/angle.json

```
@@ -25,7 +25,7 @@
               "version_added": "9"
             },
             "opera": {
-              "version_added": "15"
+              "version_added": "12"
```

This ought to be changed to `"≤12.1"` and applied to all the versions in this diff:

```suggestion
              "version_added": "≤12.1"
```